frontend/src/metabase/components/form/StandardForm.jsx

```diff
@@ -26,7 +26,7 @@ const StandardForm = ({
   resetButton = false,
   newForm = true,
   onClose = null,
-
+  submitResultDetail = null,
```

Comment:
I think it might be better to just have `formError` and `formSuccess` props that mirror `FormMessage`'s.